src/main/java/tic/tac/toe/Board.java

```diff
@@ -30,6 +31,14 @@ public void setBoard(char[] board) {
     return emptyCells;
   }
 
+  public List<Character> getCells(int[] cellsToGet) {
+    List<Character> cells = Arrays.stream(cellsToGet)
+            .boxed()
+            .map(cell -> getMarkAt(cell))
```

Comment:
I'm not positive, but I think you can do something like this:
```suggestion
            .map(this::getMarkAt)
```